### DISCUSSION TITLE
Define Workflows with Json

### DISCUSSION THREAD

**Discussion by hilkenan** [elsa-workflows/elsa-core#385] [Q&A]
Is it possible to define a Workflow with only Json? So no Coding? I not finde it at the Doumentation.

**Comment by sfmskywalker** [comment]
That is certainly possible. After all, the workflows created using the designer are also persisted as JSON, and then parsed at runtime. Others have asked about this to, so we will need to add documentation with working examples.

Until then, here is a gist that demonstrates how to load JSON into a workflow definition and how to provide those loaded definitions to the workflow runtime: https://gist.github.com/sfmskywalker/f290d2ab5a5cf7c5aa33238929b02509
